cli/src/main/java/org/eclipse/hono/cli/app/Commander.java

```diff
@@ -89,7 +92,13 @@ private void startCommandClient(final HonoConnection connection) {
                 })
                 .map(this::closeCommandClient)
                 .otherwise(error -> {
-                    LOG.error("Error sending command: {}", error.getMessage());
+                    if (ServerErrorException.extractStatusCode(error) == 503) {
```

Comment:
we usually don't compare to literal numbers, instead you should use the corresponding constant in `HttpURLConnection` ...